garage/envs/base.py

```diff
@@ -61,6 +60,7 @@ def _close_viewer_window(self):
         search in those as well.
         This method can be removed once OpenAI solves the issue.
```

Comment:
can you provide a link to an openai github issue?

Reply:
And maybe a comment about why we're importing it here (so when we read the code in the future we won't forget)

Reply:
@ryanjulian The issue has been reported to OpenAI [here](https://github.com/openai/gym/issues/1000#issuecomment-382106495), but it seems that the only change they have done from our current version 0.10.5 to 0.10.8 is [commenting](https://github.com/openai/gym/blob/70ca46a9685de333493722ee74d0cd51782b2b44/gym/envs/mujoco/mujoco_env.py#L122) (diff is [here](https://github.com/openai/gym/compare/c8f8149...70ca46a#diff-3f48026043ba85dc635adababe52094c)) the finish method in the viewer so the exception is not thrown. The guys at gym closed the issue, so if you agree I can create a new one to specifically address this concern, since the issue I referenced above comments on other things as well.
Regarding the exception I mentioned above, we're using it to detect when we have to force the closing ourselves, but for the new changes, the exception is not thrown any more and my current implementation won't close the viewer windows.
I actually tested this change (upgrading gym to v0.10.8), and the windows remain open.
If we update gym as in Keren's PR #335, we'll see the "out of memory" issue again in the cron jobs.
I will use the list of packages defined [here](https://github.com/rlworkgroup/garage/blob/f3b5ec1d97485a2f164698574c205bccebae17ec/tests/quirks.py#L90-L91) for not closing environments to detect the environments with not closing windows to avoid this dependency.
@jonashen Okay, I will add a comment to explain why the import is in not in the header.